### ISSUE TITLE
Is there a official list for Laravel middlewares?

### ISSUE THREAD

**Issue by hsluoyz** [laravel/ideas#1588]
Hi, we are developing a Laravel middleware, and want to find a place to exhibit it to Laravel users. I think the users will also have a need to find suitable middlewares in the ecosystem to solve their problem. But the only page I found is: https://laravel.com/docs/master/middleware

So I want to ask is there any official place for Laravel middleware list? Thanks.

**Comment by tomschlick** [comment]
I don't believe so, but I'm sure Taylor wouldn't mind a list being added to the documentation if you wanna craft a PR.

**Comment by mattstauffer** [comment]
It's nowhere near done but I'm building a site that's going to be an index of third-party drivers and other such things that extend Laravel (e.g. cache drivers, middleware, Socialite providers, etc.) all in one place. It'll eventually be here: https://laraveldrivers.com/

You can see a slapped-together early idea there right now. If it's something people are interested in tracking, like you're talking about here, that would probably motivate me to move faster :D

**Comment by hsluoyz** [comment]
Hi @mattstauffer , is this an official list? If yes, why not merge into the main website?

**Comment by mattstauffer** [comment]
This list is literally just one entry in each category while I slowly build the all. It’s not even close to exhaustive. 

Why not put it on the main site? Probably because Taylor doesn’t want to. If he does, he will, and then this site can go away :)
